### PR TITLE
Fixed DATEADD() numeric field overflow error

### PR DESCRIPTION
**Pull request by ZhenyeLee**
Fixed DATEADD() numeric field overflow error

Task: BABEL-3436
Signed-off-by: Zhenye Li <zhenye@amazon.com>

### Description

Fixed numeric field overflow error when casting int to numeric number. It was CAST(num AS DECIMAL(2,0)), and it failed when int number is larger than 99. Now it is CAST(num AS DECIMAL(20,10)), which can handle 10bits Integer.Add MVU file and disable the microsecond unit which is not support in SQL Server
 

### Issues Resolved

Fixed numeric field overflow error when casting int to numeric number and disable the microsecond unit which is not support in SQL Server

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by 2jungkook — DISMISSED** [review]

**Review by 2jungkook — DISMISSED** [review]

File: test/JDBC/input/BABEL-3360-vu-prepare.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+go
+select * from BABEL_3360_vu_prepare_v2;
+go
+
+create view BABEL_3360_vu_prepare_v3 as (select dateadd(millisecond, 56, null));
+go
+select * from BABEL_3360_vu_prepare_v3;
+go
+
+create view BABEL_3360_vu_prepare_v4 as (select dateadd(microsecond, 56, null));
```
Comment (surendravishnoi) [inline test/JDBC/input/BABEL-3360-vu-prepare.sql:16]: we should also call the functions from function/procedure

**Review by 2jungkook — APPROVED** [review]

**Review by 2jungkook — DISMISSED** [review]

**Review by surendravishnoi — COMMENTED** [review]
PR subject should have more context about issue fixed. Current one is too generic. Do not put any CR links here. Also, provide complete Jira id with manfred/babel prefix.

**Comment by ZhenyeLee** [comment]
> PR subject should have more context about issue fixed. Current one is too generic. Do not put any CR links here. Also, provide complete Jira id with manfred/babel prefix.

I updated the PR subject and deleted CR links. But since MANFRED is out internal project, it that okay to include in Github?

**Comment by surendravishnoi** [comment]
> > PR subject should have more context about issue fixed. Current one is too generic. Do not put any CR links here. Also, provide complete Jira id with manfred/babel prefix.
> 
> I updated the PR subject and deleted CR links. But since MANFRED is out internal project, it that okay to include in Github?

we should not include manfred here. You can open a babel jira, link it with manfred Jira and put babel Jira number here (babel-num)

**Comment by ZhenyeLee** [comment]
> > > PR subject should have more context about issue fixed. Current one is too generic. Do not put any CR links here. Also, provide complete Jira id with manfred/babel prefix.
> > 
> > 
> > I updated the PR subject and deleted CR links. But since MANFRED is out internal project, it that okay to include in Github?
> 
> we should not include manfred here. You can open a babel jira, link it with manfred Jira and put babel Jira number here (babel-num)

Updated PR Description with BABEL JIRA number, please take a look. Thanks!

File: test/JDBC/upgrade/latest/schedule [hunk 2]
```diff
@@ -89,6 +89,7 @@ BABEL-3249
 BABEL-383
 BABEL-405
 BABEL-PROCID
+BABEL-3360
```
Comment (surendravishnoi) [inline test/JDBC/upgrade/latest/schedule:92]: why only to latest schedule file? we should add it to all other upgrade sources like 13.4, 13.5 up to 14.3

**Review by surendravishnoi — COMMENTED** [review]
Pleas resolve merge conflict

**Comment by ZhenyeLee** [comment]
Created a new PR https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/497

**Pull request closed without merging**